services/eventhistory/pkg/config/config.go

```diff
@@ -0,0 +1,52 @@
+package config
+
+import (
+	"context"
+	"time"
+
+	"github.com/owncloud/ocis/v2/ocis-pkg/shared"
+)
+
+// Config combines all available configuration parts.
+type Config struct {
+	Commons *shared.Commons `yaml:"-"` // don't use this directly as configuration for a service
+
+	Service Service `yaml:"-"`
+
+	Log   *Log  `yaml:"log"`
+	Debug Debug `yaml:"debug"`
+
+	GRPC          GRPCConfig            `yaml:"grpc"`
+	GRPCClientTLS *shared.GRPCClientTLS `yaml:"grpc_client_tls"`
+
+	Events Events `yaml:"events"`
+	Store  Store  `yaml:"store"`
+
+	Context context.Context `yaml:"-"`
+}
+
+// GRPCConfig defines the available grpc configuration.
+type GRPCConfig struct {
+	Addr      string                 `ocisConfig:"addr" env:"EVENTHISTORY_GRPC_ADDR" desc:"The bind address of the GRPC service."`
+	Namespace string                 `ocisConfig:"-" yaml:"-"`
+	TLS       *shared.GRPCServiceTLS `yaml:"tls"`
+}
+
+// Store configures the store to use
+type Store struct {
+	Type         string        `yaml:"type" env:"EVENTHISTORY_STORE_TYPE" desc:"The type of the store. Supported are: 'mem', 'ocmem', 'etcd', 'redis', 'nats-js', 'noop'. See readme for details"`
+	Addresses    string        `yaml:"addresses" env:"EVENTHISTORY_STORE_ADDRESSES" desc:"A comma seperated list of addresses the store should use. No effect on inmemory implementations"`
+	Database     string        `yaml:"database" env:"EVENTHISTORY_STORE_DATABASE" desc:"The database the store should use"`
+	Table        string        `yaml:"table" env:"EVENTHISTORY_STORE_TABLE" desc:"The table the store should use"`
```

Comment:
```suggestion
	Table        string        `yaml:"table" env:"EVENTHISTORY_STORE_TABLE" desc:"The table the store should use."`
```
I guess `database table` is meant - or?